book3/14-objects.mkd

```diff
@@ -1,97 +1,100 @@
+de manera efectiva. Deberías enfocarte en este capítulo como una forma
+de aprender algunos términos y conceptos y examinar algunos ejemplos
+sencillos para sentar las bases de tu futuro aprendizaje.
 
-The key outcome of this chapter is to have a basic understanding of how
-objects are constructed and how they function and most importantly how
-we make use of the capabilities of objects that are provided to us by Python
-and Python libraries.
+El resultado clave de este capítulo será una comprensión a nivel básico
+de cómo se construyen los objetos, cómo funcionan y, lo más importante,
+cómo usar las características de los objetos que nos dan Python
+y sus librerías.
 
-Using objects
+Usando objetos
 -------------
 
-As it turns out, we have been using objects all along in this book. Python
-provides us with many built-in objects. Here is some simple code
-where the first few lines should feel very simple and natural to you.
+Curiosamente, en el libro hemos estado utilizando objetos todo este tiempo. Python
+contiene muchos objetos incluidos. He aquí un programa sencillo, cuyas
+primeras líneas deberían resultarte sumamente simples y familiares.
 
 \index{list object}
 
 \VerbatimInput{../code3/party1.py}
 
-Instead of focusing on what these lines accomplish, let's look at what is
-really happening from the point of view of object-oriented programming.
-Don't worry if the following paragraphs don't make any sense the
-first time you read them because we have not yet defined all of these terms.
+En lugar de enfocarnos en el resultado que obtienen estas líneas, enfoquémonos
+en lo que está pasando desde el punto de vista de la programación orientada
+a objetos. No te preocupes si los siguientes párrafos no parecen tener sentido
+la primera vez que los lees, pues no hemos definido todos estos términos aún.
 
-The first line *constructs* an object of type `list`, the second and third
-lines *call* the `append()` *method*, the fourth line calls
-the `sort()` method, and the fifth line *retrieves* the item at position 0.
+La primera línea *construye* un objeto de tipo 'list', la segunda y tercera
+líneas *llaman* al *método* 'append', la cuarta línea llama al método
+'sort()' y la quinta línea *recupera* el elemento en posición 0.
 
-The sixth line calls the `__getitem__()` method in the `stuff` list
-with a parameter of zero.
+La sexta línea llama al método '__getitem__()' en la lista 'stuff'
+con un parámetro de cero.
 
 ~~~~ {.python}
 print (stuff.__getitem__(0))
 ~~~~
 
-The seventh line is an even more verbose way of retrieving the 0th item
-in the list.
+La séptima línea es una manera incluso más verbosa de obtener el elemento
+en posición 0 de la lista.
 
 ~~~~ {.python}
 print (list.__getitem__(stuff,0))
 ~~~~
 
-In this code, we call the `__getitem__` method in the `list` class
-and *pass* the list and the item we want retrieved from the list
-as parameters.
+En este programa, llamamos al método '__getitem__' en la clase 'lista'
```

Comment:
When the original text has backticks, let's keep them. Sorry if I didn't explain myself on the comment about quotes and backticks on the previous review. Since this is markdown, the backticks `` are used to indicate a code statement, which renders as `this`. Just keep the format as it comes from the original chapter. Thanks, and sorry for troubling you!

Reply:
No me aproblema corregir errores referidos a la traducción, ni a los typos, etc. Sin embargo, recuerdo que para el capítulo anterior que traduje me dijiste específicamente que los backticks los cambiara por '' en todos los casos.

 El no tener claridad sobre qué es correcto y qué no me entorpece bastante la labor de traducción. Sería útil contar con una guía, aunque fuera un documento corto, para evitar estos problemas a futuro.